content/chainguard/chainguard-images/working-with-images/compiled-programs.md

```diff
@@ -0,0 +1,79 @@
+
+In comparison to the `static` and `glibc-dynamic` Chainguard Images, `gcc-glibc` is intended to be used to develop programs based on the C standard library, instead of simply hosting binaries. Because of this, it contains additional packages such as `make`, `busybox`, as well as `gcc` to compile programs.
+
+The following packages are included in the `gcc-glibc:latest` Chainguard Image:
+`binutils`, `build-base`, `busybox`, `ca-certificates-bundle`, `gcc`, `glibc`, `glibc-dev`, `glibc-locale-posix`, `gmp`, `isl`, `ld-linux`, `libatomic`, `libcrypt1`, `libgcc`, `libgo`, `libgomp`, `libstdc++`, `libstdc++-dev`, `libxcrypt`, `libxcrypt-dev`, `linux-headers`, `make`, `mpc`, `mpfr`, `nss-db`, `nss-hesiod`, `pkgconf`, `posix-cc-wrappers`, `wolfi-baselayout`, and `zlib`.
+
+### glibc-openssl
+*Paid Image* \
+The `glibc-openssl` Chainguard Image is designed for C applications which depend on OpenSSL. It contains a few packages in addition to those found in the `glibc-dynamic` image to achieve this. It comes in two variants,`latest` and `latest-dev`. As in the aforementioned images, `latest` is designed for deployment, while `latest-dev` contains additional packages to assist in program development such as a shell and package manager.
+
+The following packages are included in the `glibc-openssl:latest` Chainguard Image:
+`ca-certificates-bundle`, `chainguard-baselayout`, `glibc`, `glibc-locale-posix`, `ld-linux`, `libcrypto3`, `libgcc`, `libssl3`, `libstdc++`, `openssl`, `openssl-provider-legacy`, and `wolfi-baselayout`.
+
+### wolfi-base
+The `wolfi-base` image is a minimal container image based on the [Wolfi *un-distro*](https://github.com/wolfi-dev/), a community-oriented Linux distribution created by Chainguard to facilitate image builds. The `wolfi-base` image contains `busybox` and `apk-tools` so that you can build your own packages for a custom image. It supports `glibc`.
+
+The following packages are included in the `wolfi-base:latest` Chainguard Image:
+`apk-tools`, `busybox`, `ca-certificates-bundle`, `chainguard-baselayout`, `glibc`, `glibc-locale-posix`, `ld-linux`, `libcrypt1`, `libcrypto3`, `libssl3`, `libxcrypt`, `wolfi-base`, `wolfi-baselayout`, `wolfi-keys`, and `zlib`.
+
+### chainguard-base
+*Paid Image* \
+In addition to the functionality of the `wolfi-base` image, `chainguard-base` reports as being a Chainguard Image, which scanners use to determine what security feeds to reference for vulnerabilities. Additionally, the `chainguard-base` image provides access to vulnerability remediation SLAs to ensure your containers are always up-to-date with the latest releases and patches.
+
+The following packages are included in the `chainguard-base:latest` Chainguard Image:
+`apk-tools`, `busybox`, `ca-certificates-bundle`, `chainguard-baselayout`, `glibc`, `glibc-locale-posix`, `ld-linux`, `libcrypt1`, `libcrypto3`, `libssl3`, `libxcrypt`, `wolfi-base`, `wolfi-baselayout`, `wolfi-keys`, and `zlib`.
+
+
+## What about musl?
```

Comment:
You can make muscle monospace in the title (I know it's kinda weird but that's what we've settled on).